### PR TITLE
fix(workspaces): modify notifications [FF]

### PR DESCRIPTION
**Pull request by vapurrmaid**
Starting in v1.29, the Notifications preference page is removed.

### CODE REVIEW

File: workspaces/preferences.md [hunk 1]
```diff
@@ -68,8 +65,6 @@ Please note that:
 - You must enable native notifications for each browser on which you run Coder
   Enterprise
 
-You can check whether notifications are enabled and working by click **Test**.
```
Comment (vapurrmaid) [inline workspaces/preferences.md:71]: PS: there was a type here (click -> clicking), if we want to backport that to current versions.

**Comment by vapurrmaid** [comment]
Marked as draft, awaiting `v1.29` branch

**Comment by vapurrmaid** [comment]
@khorne3 is this safe to merge into `main`, or should it wait? I think we switched over to tag-based, so that it's safe to merge into `main`, but I want to be sure

**Comment by khorne3** [comment]
@vapurrmaid let's wait for 1.29

**Comment by vapurrmaid** [comment]
Thanks for putting this into releas @khorne3 🙌🏻 !!

**Pull request merged**